### PR TITLE
In PTL tests, removed integer typecasting of starttime and endtime in log_match and log_parse

### PR DESCRIPTION
**Pull request by birbalsain**
#### Describe Bug or Feature
Some logs missed because of integer typecast of stattime and endtime in log_match and log_parser

#### Describe Your Change
Removed integer typecasting of starttime and endtime in log_match and log_parser

#### Attach Test and Valgrind Logs/Output

time_typecast1
time_typecast

### CODE REVIEW

**Comment by visheshh** [comment]
Hi @birbalsain , Thanks for making the changes. Please attach test logs for all the tests you have modified.

**Comment by birbalsain** [comment]
@visheshh I have attached the log.

File: test/fw/bin/pbs_loganalyzer [hunk 1]
```diff
@@ -151,7 +153,7 @@ if __name__ == '__main__':
                    "re-conditional=", "estimated-info-only", "silent",
                    "db-type=", "db-access="]
         opts, args = getopt.getopt(sys.argv[1:], shortopt, longopt)
-    except:
+    except BaseException:
```
Comment (bhroam) [inline test/fw/bin/pbs_loganalyzer]: The correct answer for blanket excpets is to determine what exceptions are expected.  Doing this is no better than a blanket except.  BaseException includes things like KeyboardInterrupt which is undesired.  Until we go through to fix all of these at once, I'd leave them alone.

**Comment by birbalsain** [comment]
@bhroam I have done the requested changes.

**Comment by bhroam** [comment]
@birbalsain you haven't fixed all instances of reservations.  Please rereview your entire PR and don't remove the int() casting when submitting reservations.

**Comment by birbalsain** [comment]
@bhroam at some places same time is used by reservation and log match. I did the changes for them.  Are you saying about them? if yes then should i remove changes?

**Comment by birbalsain** [comment]
@PBSPro/pbspro-maintainers Please take this PR further.

**Pull request merged**